shared/python/slack_util.py

```diff
@@ -8,11 +8,18 @@
 import requests
 import sys
 
+from slack import WebClient
+from slack.errors import SlackApiError
 from common import render_exception
 
 def generate_ping_list(user_ids):
     return ', '.join(['<@{}>'.format(user_id) for user_id in user_ids])
 
+def new_client():
+    try:
+        return True, 'success', WebClient(token=os.environ['SLACK_CLIENT_TOKEN'])
```

Comment:
I think this would be better to have in the config

Reply:
We can do that if we don't upload config to the codebase, since anyone has access to the token can take control of the App.

Reply:
That is a valid concern. We shouldn't _commit_ our API token (just as we don't commit webhooks) but I think it should be kept in a config file to keep the config manageable and easily reproducible.